### PR TITLE
chore: Setup dependabot

### PR DESCRIPTION
**Pull request by atomrc**

### CODE REVIEW

**Review by tlebon — APPROVED** [review]

**Comment by tlebon** [comment]
i suggest updating auto-merge to make sure we don't wake up to 100 PRs on monday

File: .github/auto-merge.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 - match:
     dependency_type: all
-    update_type: 'semver:major'
+    update_type: 'semver:minor'
```
Comment (tlebon) [inline .github/auto-merge.yml:3]: i guess patch also

Comment (tlebon) [reply .github/auto-merge.yml:3]: some nice ideas here, similar to our other auto-merge.yml files https://github.com/marketplace/actions/enable-github-automerge

Comment (atomrc) [reply .github/auto-merge.yml:3]: Oh I'd expect `minor` to also include what is smaller that minor (so to include `patch` in it). Isn't it?

Comment (atomrc) [reply .github/auto-merge.yml:3]: Oh it does include `patch`
image
(cf. https://github.com/ahmadnassri/action-dependabot-auto-merge#configuration-file-syntax)

**Pull request merged**